### ISSUE TITLE
OCA aircraft strike missions

### ISSUE THREAD

**Issue by DanAlbert** [dcs-liberation/dcs_liberation#349]

**Comment by DanAlbert** [comment]
Split https://github.com/Khopa/dcs_liberation/issues/432 to cover runway bombing. This will cover attacking grounded aircraft.

**Comment by DanAlbert** [comment]
It doesn't appear to be possible to task the AI to do this. We can task them with individual bombing locations on specific parked aircraft, but that means that a 4-ship will take many passes dropping one bomb each time.

Until https://forums.eagle.ru/forum/english/digital-combat-simulator/dcs-world-2-5/dcs-wishlist-aa/7149996-task-for-attacking-aircraft-on-the-ground is fixed there's no good way to do this. We could add the mission type just for the player, but I don't think there's any reason to do so since https://github.com/Khopa/dcs_liberation/issues/432 would have an identical flight plan.

**Comment by DanAlbert** [comment]
@Khopa played with it a bit more and was able to get it working. https://forums.eagle.ru/forum/english/digital-combat-simulator/dcs-world-2-5/bugs-and-problems-ai/ai-ad/7150830-search-then-engage-in-zone-does-not-search explains a bit.

We need to make a flight plan that overflies the airfield because the zone for Search Then Engage in Zone isn't a zone to search. It's a filter for the units found in the search _along the route_.

AI behavior here isn't great either. They don't seem to see very well so they can't begin their attack until it's almost too late. The best behavior seems to happen with aircraft flying well below our usual attack altitudes. I guess the AI doesn't use its targeting pod to search?
